core/src/main/java/data/people/PeopleRepository.kt

```diff
@@ -0,0 +1,26 @@
+package com.example.core.data.people
+
+import com.example.core.data.people.datasource.PeopleLocalDataSource
+import com.example.core.data.people.datasource.PeopleRemoteDataSource
+import com.example.model.PopularActor
+import kotlinx.coroutines.flow.flow
+import javax.inject.Inject
+
+class PeopleRepository @Inject constructor(
+    private val remote: PeopleRemoteDataSource,
+    private val local: PeopleLocalDataSource
+){
+
+    //------------ Popular capabilities ------------
+
+    fun observePopularPeople() = local.pPeopleStore.observeEntries()
```

Comment:
pPeopleStore is not good naming
should be camel case start with lower case letter